lib/kvdb/kvdb_rparams.c

```diff
@@ -968,6 +968,7 @@ static const struct param_spec pspecs[] = {
         .ps_flags = 0,
```

Comment:
Why is ps_size added only to a selected set of parameters?

Reply:
Most of the parameters already had .ps_size applied, but a few didn't because it didn't make any sense, but I am using this for my introspection PR and I felt it was generally a good change, so I pulled it out.